lib/index.d.ts

```diff
@@ -39,6 +36,9 @@ type Headers = {
         | undefined;
 };
 
+
+interface QueryParams extends QueryString.ParsedUrlQueryInput {}
```

Comment:
move this before where it's used

Reply:
This isn't compatible with the old definition:
`{[key: string]: string | string[]};`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/querystring.d.ts#L23
`NodeJS.Dict<string | number | boolean | ReadonlyArray<string> | ReadonlyArray<number> | ReadonlyArray<boolean> | null> {}`

Reply:
Thanks! I am making this type more generic and correctly respect the implementation
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/querystring.d.ts#L60

Also, I have the following quick test on the compatibility
```
import querystring from 'querystring';
const a: {[key: string]: string | string[]}  = { b: '1' };
const b: querystring.ParsedUrlQueryInput = a;
```

Please let me know if anything I missed

Reply:
This looks fine to me 👍  But I'd reference `ParsedUrlQueryInput` where it's used instead of creating a new, empty type.